### PR TITLE
Publish version 0.0.6

### LINKED ISSUE #145: Publish an updated version of tasit-sdk, tasit-account, and tasit-action

### PR DESCRIPTION
**Pull request by pcowgill**
### Issue link

https://github.com/tasitlabs/TasitSDK/issues/145

### Auto-close the issue?

Closes #145 

### Types of changes

Bug fix (non-breaking change that fixes an issue)

New feature (non-breaking change that adds functionality)

Technical debt (a code change that doesn't fix a bug or add a feature but makes something clearer for devs)

### CODE REVIEW

File: packages/tasit-account/package.json [hunk 1]
```diff
@@ -35,5 +35,6 @@
     "@babel/register": "^7.0.0",
     "chai": "^4.2.0",
     "mocha": "^5.2.0"
-  }
+  },
+  "gitHead": "e02f70a8c70721039e2c88d7b55d89f50c72e028"
```
Comment (pcowgill) [inline packages/tasit-account/package.json]: Hmm, it's interesting that this ended up here

Comment (pcowgill) [reply packages/tasit-account/package.json]: Or is this just a new feature of npm I didn't know about?

Comment (marcelomorgado) [reply packages/tasit-account/package.json]: https://github.com/lerna/lerna/issues/1880#issuecomment-456160357
> gitHead is a temporary property only meant to exist during lerna publish when it is constructing the package tarballs. I don't know why it is appearing in your use case.

Comment (pcowgill) [reply packages/tasit-account/package.json]: Good catch. It’s probably from the lerna publish failures. I’ll remove it before we merge this

**Comment by pcowgill** [comment]
@marcelomorgado I'll merge this once you review the latest changes. Thanks!

**Pull request merged**